artmanflow/web/templates/java_sources_staging.yaml

```diff
@@ -0,0 +1,14 @@
+execution_id: ''
```

Comment:
It doesn't seem quite right for a configuration yaml file to live under a templates/ directory.